src/types/user_operation.rs

```diff
@@ -5,25 +5,32 @@ use ethers::types::{Address, Bytes, TransactionReceipt, H256, U256};
 use ethers::utils::keccak256;
 use serde::{Deserialize, Serialize};
 use std::ops::Deref;
-use std::str::FromStr;
 
 pub type UserOperationHash = H256;
 
-#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, EthAbiCodec, EthAbiType)]
-#[serde(rename_all = "camelCase")]
-pub struct UserOperation {
-    pub sender: Address,
-    pub nonce: U256,
-    pub init_code: Bytes,
-    pub call_data: Bytes,
-    pub call_gas_limit: U256,
-    pub verification_gas_limit: U256,
-    pub pre_verification_gas: U256,
-    pub max_fee_per_gas: U256,
-    pub max_priority_fee_per_gas: U256,
-    pub paymaster_and_data: Bytes,
-    pub signature: Bytes,
+// In order to mute unwrap used lint error which is generated by the #derive(EthAbiType)
+// UserOperation is put in the private mod and mute the error.
+#[allow(clippy::unwrap_used)]
```

Comment:
EthAbiType would generate unwrap which I could not avoid now.

Reply:
https://github.com/gakonst/ethers-rs/issues/2053 created an issue for ethers-rs

Reply:
@zsluedem Do you want to wait with this PR to see if this fix in `ethers-rs` will be merged in the coming days (we can then just update the lib version)?

Reply:
Hm.. I am fine with either way.

Reply:
Then let's wait.

Reply:
@Vid201 Updated now.